tests/primer/test_primer_external.py

```diff
@@ -0,0 +1,66 @@
+# Licensed under the GPL: https://www.gnu.org/licenses/old-licenses/gpl-2.0.html
+# For details: https://github.com/PyCQA/pylint/blob/main/LICENSE
+import logging
+import subprocess
+from pathlib import Path
+
+import pytest
+from pytest import LogCaptureFixture
+
+from pylint.testutils.primer import PackageToLint
+
+PRIMER_DIRECTORY = Path(".pylint_primer_tests/").resolve()
+PACKAGES_TO_LINT = {
+    "black": PackageToLint(
+        url="https://github.com/psf/black.git",
+        branch="main",
+        directories="src tests",
+    ),
+    "home-assistant": PackageToLint(
+        url="https://github.com/home-assistant/core.git",
+        branch="dev",
+        directories="homeassistant",
+    ),
+    "graph-explorer": PackageToLint(
+        url="https://github.com/vimeo/graph-explorer.git",
+        branch="master",
+        directories="graph_explorer",
+        pylintrc_relpath=".pylintrc",
+    ),
+    "pygame": PackageToLint(
+        url="https://github.com/pygame/pygame.git",
+        branch="main",
+        directories="src_py",
+    ),
+}
+"""Dictionary of external packages used during the primer test"""
+
+
+class TestPrimer:
+    @staticmethod
+    @pytest.mark.primer
+    @pytest.mark.parametrize("package", PACKAGES_TO_LINT.values(), ids=PACKAGES_TO_LINT)
+    def test_primer_external_packages_no_crash(
+        package: PackageToLint,
+        caplog: LogCaptureFixture,
+    ) -> None:
+        """Runs pylint over external packages to check for crashes and fatal messages
+
+        We only check for crashes (bit-encoded exit code 32) and
+        fatal messages (bit-encoded exit code 1). We assume that these external repositories
+        do not have any fatal errors in their code so that any fatal errors are pylint false
+        positives
+        """
+        caplog.set_level(logging.INFO)
+        package.lazy_clone()
+        try:
+            command = ["pylint"] + package.paths_to_lint + package.pylint_args
```

Comment:
```suggestion
            # We want to test all the code we can
            command = ["pylint", "--enable-all-extensions", "--enable=all"] + package.paths_to_lint + package.pylint_args
```

This is what I meant with the other comment. I feel like in the other file it might go unnoticed sooner. This clearly demonstrates the intent of these tests.

Reply:
Ho sorry I misunderstood, of course you're right.